README.md

```diff
@@ -0,0 +1,13 @@
+## glass-website
```

Comment:
```suggestion
<div align="center">

# Glass Website

*This website shows an example of glassmorphism (a trending topic for web developers)*

<br>

</div>
```

Reply:
It will center the Heading and add a short description there's no need to greet contributors and then add a description.